### PR TITLE
World building and character creation

### PR DESCRIPTION
**Pull request by agude**
Starting to work on these.

I'm trying to write short sentences, but I think I'm too committed to it. It feels unnatural in some places.

How many examples do we want in the text? They make it longer but I think it feels really sterile without them.

### CODE REVIEW

File: _chapters/00-outline.md [hunk 1]
```diff
@@ -69,11 +69,49 @@ Not 100% sure about where to introduce aspects, but I think the flow here makes
 
 ## World Building
 
+<!--
+QUESTION:
+I keep saying game, setting, world.
+Do we want one word for these? World aspects? Setting aspects?
+-->
+
+A game's setting helps players understand how their characters fit into the world.
```
Comment (agude) [inline _chapters/00-outline.md:78]: I need an intro and don't have one yet.

File: _chapters/00-outline.md [hunk 2]
```diff
@@ -69,11 +69,49 @@ Not 100% sure about where to introduce aspects, but I think the flow here makes
...
+Write the aspects down somewhere everyone can see them.
+Don't worry about making them perfect as they're just a starting point.
+As the story progresses the setting may change; when this happens feel free to add or modify setting aspects.
+
 ## Character Creation
 
-do this as a character sheet walkthrough
-discuss concept and trouble, since you set those right away
-declare the rest (stress, consequences, refresh?) but define them later
+Player characters (PCs) are mechanically simple but narratively diverse.
```
Comment (agude) [inline _chapters/00-outline.md:93]: Likewise don't love this intro. =

File: _chapters/00-outline.md [hunk 3]
```diff
@@ -69,11 +69,49 @@ Not 100% sure about where to introduce aspects, but I think the flow here makes
...
+A PC's trouble aspect describes the sort of things that get in the PC's way and make their story interesting.
+It does not need to be the most likely thing to cause the PC trouble.
+It is a good way for the player to indicate what sort of problems they would find interesting to deal with.
+
+Some example troubles are:
+~~I Don't Know Which Side I'm On~~,
+~~Everyone I Get Close To Ends Up In Trouble~~,
+or ~~My Past Is A Dangerous Mystery; Even to Me~~.
+
+Neither PC aspect is entirely good or entirely bad.
```
Comment (agude) [inline _chapters/00-outline.md:112]: Holy shit I use PC a lot.

File: _chapters/00-outline.md [hunk 4]
```diff
@@ -69,11 +69,49 @@ Not 100% sure about where to introduce aspects, but I think the flow here makes
...
+Just like everything else in {GameName}, the setting is defined by its aspects.
+Start with two aspects to describe the world: a genre and a conflict.
+The genre aspect gives a sense of what, when, and where the story takes place.
+The conflict aspect introduces the who and the why of the setting's biggest source of dramatic tension.
+
+Write the aspects down somewhere everyone can see them.
+Don't worry about making them perfect as they're just a starting point.
+As the story progresses the setting may change; when this happens feel free to add or modify setting aspects.
+
 ## Character Creation
```
Comment (agude) [inline _chapters/00-outline.md:91]: Realize I don't even talk about the mechanics. Does that go here or do we have a "How to do stuff and roll dice" section?

**Review by agude — COMMENTED** [review]
Add some line comments for the reviewer.

File: _chapters/00-outline.md (hunk 4 above)
Comment (charles-uno) [reply _chapters/00-outline.md:104]: I think that's later. probably ok for this to lean on the previous section, which defines aspects and indicates that they are important to whatever degree

File: _chapters/00-outline.md [hunk 5]
```diff
@@ -69,11 +69,62 @@ Not 100% sure about where to introduce aspects, but I think the flow here makes
...
+or ~~Backwater Farm Boy Looking to Join The Academy~~.
+
+A PC's trouble aspect describes the sort of things that get in the PC's way and make their story interesting.
+It does not need to be the most likely thing to cause the PC trouble.
+It is a good way for the player to indicate what sort of problems they would find interesting to deal with.
+
+Some example troubles are:
+~~I Don't Know Which Side I'm On~~,
+~~Everyone I Get Close To Ends Up In Trouble~~,
+or ~~My Past Is A Dangerous Mystery; Even to Me~~.
```
Comment (charles-uno) [inline _chapters/00-outline.md:123]:
```suggestion
or ~~My Past Is A Dangerous Mystery, Even to Me~~.
```

Comment (charles-uno) [reply _chapters/00-outline.md:123]: em dash would also be ok

**Review by charles-uno — APPROVED** [review]
Feels like this is mostly pulling the stuff we like from the previous iteration. should we also nuke the old version to reduce duplicate text?

**Comment by charles-uno** [comment]
Examples baked into the prose vs blockquote examples? Is it weird to have a mix?

Approved because I think pulling some stuff over from the old text is a fine place to pause before writing "real" paragraphs

**Comment by agude** [comment]
> Examples baked into the prose vs blockquote examples? Is it weird to have a mix?
> 
> Approved because I think pulling some stuff over from the old text is a fine place to pause before writing "real" paragraphs

The new version has a lot of "examples in the prose" and "block quote examples".

I think it's fine to have both. The main difference is:

- **In the prose**: Short examples, often like an example aspect or something general. For example: "A bad aspect is `I'm a loser` because it is vague and doesn't add character depth." or "A `Firemage cast out at birth` can attack using fireballs, but his friend the `Worlds best archer; when I'm not drunk`would use arrows."
- Block quotes should be longer play examples from our imaginary setting, and probably involve dialogue or the players themselves.

**Pull request merged**